backend/api.py

```diff
@@ -240,10 +240,14 @@ def niceties_from_me():
 def niceties_for_me():
     ret = []
     whoami = current_user().id
-    valid_niceties = (Nicety.query
-                      .filter(Nicety.end_date + timedelta(days=1) < datetime.now())  # show niceties one day after the end date
-                      .filter(Nicety.target_id == whoami)
-                      .all())
+    if app.config.get("DEV") == "TRUE":
+        valid_niceties = (Nicety.query
+                          .all())
```

Comment:
Yes - using my local test DB, I only have niceties I wrote for others, and that seemed like the simplest way to get some data for the Niceties About Me page. I haven't dug into the mock/fixture data that might address this, or tried writing directly to the DB to create some fake Niceties About Me, but I can if you like. Or keep this a local change, as it's primarily for my convenience. What do you prefer?

Reply:
Ah, that makes sense! Seems useful, but maybe we could hide it behind a feature flag - perhaps an environment variable like `DEBUG_SHOW_ALL` or so?

Reply:
I can switch it to a flag called `DEBUG`. I made a similar call in `backend/util.py/admin_access` that sets the user to admin (along with some hardcoded user IDs!) which I would also change to `DEBUG`, leaving `DEV` only used in `backend/auth.py/login`. I've been used to thinking of `DEV` as functionally equivalent with `DEBUG` but an extra line in .env is no big deal. Does that sound good?

Reply:
I saw the backend change and liked it; making the developer an admin is very useful!

I think the distinction in my mind is adding functionality (or granting permissions) vs changing functionality. Showing all niceties is a feature that is useful for debugging, so I would like to include it! Since it's not representative of how the system will behave for any user in production, I think it would be helpful to signal that to developers and have the option to toggle it off and on separately.

As an aside, I worked out how to add niceties for debugging, as I was working on other branches without this change:

```sql
INSERT INTO nicety (end_date, author_id, target_id, anonymous, faculty_reviewed, text, no_read, date_updated)
VALUES ('2017-03-23', 2092, 2092, true, false, encode('Niceties are base64 encoded!', 'base64'), false, now());
```

See https://www.postgresql.org/docs/current/functions-binarystring.html

Reply:
Sorry, that was a lot of words without a clear summary at the end!

I like this feature, I'd like it to be behind a feature flag, but I don't want to tie it to an existing feature flag or to a feature flag used by any other features. I'd like the feature flag to be clearly named, but am totally fine with whatever you come up with. How does that sound?

Reply:
I used your suggestion of `DEBUG_SHOW_ALL` in commit 63aa1ba. It's set to False if unspecified by the user. Let me know if the README is clear enough.

Reply:
Awesome! Thank you, @christalee, that looks great.